talisker/logs.py

```diff
@@ -323,8 +323,8 @@ def format(self, record):
                                                'replace')
         return s
 
-    def escape_quotes(self, s):
-        return s.replace('"', '\\"')
+    def clean_message(self, s):
+        return s.replace('"', '\\"').replace('\n', '')
```

Comment:
Solving a similar problem with syslog in the past, we replaced \n with `#012`

I'm a bit concerned that just throwing it away will cause misleading logs - if we had something blow up on newlines, how would we be able to identify that request from these logs?